<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMainForm</class>
 <widget class="QWidget" name="QMainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tetris</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QFrame" name="frameFigure">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>180</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="figureArea">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>140</width>
          <height>140</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonPrev">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;-</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonSig">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>70</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>-&gt;</string>
        </property>
       </widget>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="labelPuntuacion">
       <property name="text">
        <string>Puntuación</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLCDNumber" name="lcdPuntuacion"/>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="labelTCaida">
       <property name="text">
        <string>Tiempo de caída (sg)</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QDoubleSpinBox" name="spinBoxFallTime">
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>5.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QGroupBox" name="gBAddRows">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>120</height>
        </size>
       </property>
       <property name="title">
        <string>Añadir filas</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <widget class="QSpinBox" name="spinFromFigs">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinEveryFigs">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
       <widget class="QLabel" name="labelFromFigs1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>A partir de</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelFromFigs2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>30</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>piezas</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelEveryFigs2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>80</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>piezas</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelEveryFigs1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Cada</string>
        </property>
       </widget>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QFrame" name="frameControls">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>120</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="buttonGirar">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Girar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonIzquierda">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Izda</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonDerecha">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>10</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Dcha</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonAbajo">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Abajo</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonComenzar">
       <property name="text">
        <string>Comenzar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonPausar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Pausar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
      <widget class="QFrame" name="containerArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>420</width>
         <height>595</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>420</width>
         <height>595</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBoxPruebas">
       <property name="title">
        <string>Modo de pruebas</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <widget class="QPushButton" name="buttonVaciar">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Vaciar contenedor</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonBorrarFilas">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>80</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Borrar filas</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonAnnadirF">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>80</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Añadir fila</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonInsertarF">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Insertar figura</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonBorrarF">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>40</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Borrar figura</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonHuecoLibre">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>40</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Hueco libre</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>groupBoxPruebas</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonPrev</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBoxPruebas</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonSig</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>661</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBoxPruebas</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonComenzar</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>628</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>577</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBoxPruebas</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonPausar</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>679</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>598</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBoxPruebas</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonIzquierda</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>655</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBoxPruebas</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonDerecha</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>676</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBoxPruebas</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonAbajo</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>686</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
